aicsimageio/aics_image.py

```diff
@@ -305,6 +305,16 @@ def metadata(self) -> Any:
         # The reader can implement read optimization or not.
         return self.reader.metadata
 
+    @property
+    def get_metadata_as_ome(self):
+        """
+
+        Returns
+        -------
+
+        """
+        return self.reader.get_metadata_as_ome
```

Comment:
Similarly, once the reader function is converted to a function not a property you will need to add a function call here `()`